src/commands/Anime/manga.ts

```diff
@@ -72,6 +51,34 @@ export default class extends SkyraCommand {
 		return total ? ((total / (max * 20)) * 100).toFixed(2) : '--.--';
 	}
 
+	private buildDisplay(entries: (Kitsu.Datum<Kitsu.MangaAttributes>)[], message: KlasaMessage) {
+		const display = new UserRichDisplay();
+
+		for (const entry of entries) {
+			this.client.console.log(entry.attributes);
+			this.client.console.error('===========');
```

Comment:
👀

Reply:
woops LOL

Reply:
it'll be gone after the algolia rewrite dw